CMakeLists.txt

```diff
@@ -320,6 +330,15 @@ message("  bitcoin-cli ......................... ${BUILD_CLI}")
 message("  bitcoin-tx .......................... ${BUILD_TX}")
 message("  bitcoin-util ........................ ${BUILD_UTIL}")
 message("  bitcoin-wallet ...................... ${BUILD_WALLET_TOOL}")
+message("Libraries:")
+if(BUILD_SHARED_LIBS)
```

Comment:
Couldn't this be collapsed down to just:
```txt
if(BUILD_SHARED_LIBS)
  message("  library type ........................ Shared")
else()
  message("  library type ........................ Static")
endif()
```

Then there's no need for an intermediate variable, and "cleaning up" afterwards? We know there's only two types of libs we are building.

Reply:
Thanks! Done.

Reply:
This implies that when `BUILD_SHARED_LIBS=1` then no static libraries will be built. Is this the case? And no plan to build both in the future? I think it is normal to provide both static and shared libraries, for example:

```
	/usr/local/lib/libx264.a
	/usr/local/lib/libx264.so
	/usr/local/lib/libx264.so.164
...
	/usr/local/lib/libidn.a
	/usr/local/lib/libidn.so
	/usr/local/lib/libidn.so.12
	/usr/local/lib/libidn.so.12.6.3
...
	/usr/local/lib/libssh2.a
	/usr/local/lib/libssh2.so
	/usr/local/lib/libssh2.so.1
	/usr/local/lib/libssh2.so.1.0.1
```

Reply:
> And no plan to build both in the future?

Correct. No plans to build a shared lib and a static lib simultaneously, in a single run. Please see the related discussions:
- https://github.com/hebasto/bitcoin/pull/32#discussion_r1366094140
- https://github.com/bitcoin-core/secp256k1/pull/1230

Also, you might want to look at https://github.com/bitcoin/bitcoin/issues/28779.